lively.morphic/morph.js

```diff
@@ -2509,6 +2525,26 @@ export class Morph {
     this.stopScripts(this._tickingScripts.filter(ea => ea.equals(script)));
   }
 
+  // -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+  // comments
+  // -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+  async addComment (commentText, relativePosition = pt(0, 0)) {
+    const comment = new Comment(commentText, relativePosition);
+    this.comments.push(comment);
+    await CommentBrowser.addCommentForMorph(comment, this);
+    return comment;
+  }
+
+  async removeComment (commentToRemove) {
+    this.comments = this.comments.filter(comment => !commentToRemove.equals(comment));
+    CommentBrowser.removeCommentForMorph(commentToRemove, this);
+  }
+
+  emptyComments () {
+    this.comments.forEach((comment) => CommentBrowser.removeCommentForMorph(comment, this));
+    this.comments = [];
+  }
+
```

Comment:
Since you say that morph should be as concise as possible, this could also be moved to Comment with static methods that take a morph as an argument. However this may make the methods somewhat harder to understand and these methods would then be responsible for the comments field on some foreign morph. What is your opinion on this, @merryman ?

Reply:
Fine for now